src/main/java/co/norse/chat/User.java

```diff
@@ -0,0 +1,24 @@
+package co.norse.chat;
+
+public class User {
```

Comment:
add default constructor, add getters and setters, also generate hashCode and equals.